torch/onnx/symbolic_opset9.py

```diff
@@ -1917,6 +1917,12 @@ def log_softmax(g, input, dim, dtype=None):
     return return_op
 
 
+@symbolic_helper.parse_args("v", "i", "i")
+def _log_softmax(g, input, dim, half_to_float):
+    # TODO: Handle half_to_float.
```

Comment:
If this is not resolved in this PR, please open an issue in PyTorch to track it, and mention it here. e.g. # TODO(#ISSUE_NUMBER): ...

Reply:
Sure. #81939 is created for tracking.